src/core/components/appStore/Slice.tsx

```diff
@@ -0,0 +1,41 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { ProviderI } from "../ProviderModal";
+
+interface CommonState {
+    selectedProvider: ProviderI;
+    providerAuthURL: string;
+    providerList: [];
+    configOpen: boolean,
+    providerOpen: boolean
+}
+
+const commonSlice = createSlice({
```

Comment:
Any reason on why we have it as slice?

Reply:
Slice is used to access and manage data across multiple components.